admin/configuration.md

```diff
@@ -5,27 +5,40 @@ The configuration file of TheHive is `/etc/thehive/application.conf` by default.
 You can have a look at the [default settings](default-configuration.md).
 
 ## Table of Contents
-  * [1\. Database](#1-database)
-  * [2\. Datastore](#2-datastore)
-  * [3\. Authentication](#3-authentication)
-    * [3\.1 LDAP/AD](#31-ldapad)
-    * [3\.2 OAuth2/OpenID Connect](#32-oauth2openid-connect)
-  * [4\. Streaming (a\.k\.a The Flow)](#4-streaming-aka-the-flow)
-  * [5\. Entity size limit](#5-entity-size-limit)
-  * [6\. Cortex](#6-cortex)
-  * [7\. MISP](#7-misp)
-    * [7\.1 Configuration](#71-configuration)
-    * [7\.2 Associate a Case Template to Alerts corresponding to MISP events](#72-associate-a-case-template-to-alerts-corresponding-to-misp-events)
-    * [7\.3 Event Filters](#73-event-filters)
-  * [7\.4 MISP Purpose](#74-misp-purpose)
-  * [8\. HTTP Client Configuration](#8-http-client-configuration)
-  * [9\. Monitoring and Performance Metrics (deprecated)](#9-monitoring-and-performance-metrics-deprecated)
-  * [10\. HTTPS](#10-https)
-    * [10\.1 HTTPS using a reverse proxy](#101-https-using-a-reverse-proxy)
-    * [10\.2 HTTPS without reverse proxy](#102-https-without-reverse-proxy)
-    * [10\.3 Strengthen security](#103-strengthen-security)
-
-## 1. Database
+  * [1\. Secret](#1-secret-key)
+  * [2\. Database](#2-database)
+  * [3\. Datastore](#3-datastore)
+  * [4\. Authentication](#4-authentication)
+    * [4\.1 LDAP/AD](#41-ldapad)
+    * [4\.2 OAuth2/OpenID Connect](#42-oauth2openid-connect)
+  * [5\. Streaming (a\.k\.a The Flow)](#5-streaming-aka-the-flow)
+  * [6\. Entity size limit](#6-entity-size-limit)
+  * [7\. Cortex](#7-cortex)
+  * [8\. MISP](#8-misp)
+    * [8\.1 Configuration](#81-configuration)
+    * [8\.2 Associate a Case Template to Alerts corresponding to MISP events](#82-associate-a-case-template-to-alerts-corresponding-to-misp-events)
+    * [8\.3 Event Filters](#83-event-filters)
+    * [8\.4 MISP Purpose](#84-misp-purpose)
+  * [9\. HTTP Client Configuration](#9-http-client-configuration)
+  * [10\. Monitoring and Performance Metrics (deprecated)](#10-monitoring-and-performance-metrics-deprecated)
+  * [11\. HTTPS](#11-https)
+    * [11\.1 HTTPS using a reverse proxy](#111-https-using-a-reverse-proxy)
+    * [11\.2 HTTPS without reverse proxy](#112-https-without-reverse-proxy)
+    * [11\.3 Strengthen security](#113-strengthen-security)
+
+## 1. Secret Key
+
+The only required parameter in order to start TheHive is the key of the server (`play.http.secret.key`). This key is used to authenticate cookies that contain data. If TheHive runs in cluster mode, all instances must share the same key.
+
+You can set a new random secret to the initial config using the following command:
+
+```shell
+sudo sed -i "s/^#play\.http\.secret\.key=.*/play.http.secret.key = \"$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | head -c 64)\"/" /etc/thehive/application.conf
+```
+
+**NOTE: This secret is not autogenerated during installation via package managers (apt, rpm, etc).**
```

Comment:
I don't know if this is 100% true - I am basing it off my experience with  the most recent Debian package (`3.4.0-1`).